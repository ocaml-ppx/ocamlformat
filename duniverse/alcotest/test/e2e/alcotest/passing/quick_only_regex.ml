let () =
  let open Alcotest in
  let id () = () in
  run
    ~argv:[| ""; "test"; "--quick"; ".*-a" |]
    __FILE__
    [
      ( "test-a",
        [
          test_case "Quick & passes filter" `Quick id;
          test_case "Slow & passes filter" `Slow id;
        ] );
      ( "test-b",
        [
          test_case "Slow & fails filter" `Slow id;
          test_case "Quick & fails filter" `Quick id;
        ] );
    ]
