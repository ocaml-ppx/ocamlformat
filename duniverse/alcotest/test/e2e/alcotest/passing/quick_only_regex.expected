Testing `test/e2e/alcotest/passing/quick_only_regex.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Quick & passes filter.
  [SKIP]        test-a          1   Slow & passes filter.
  [SKIP]        test-b          0   Slow & fails filter.
  [SKIP]        test-b          1   Quick & fails filter.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 1 test run.
